### ISSUE TITLE
Feature Request: Add sigma() method

### ISSUE THREAD

**Issue by billdenney** [nlmixrdevelopment/nlmixr#415]
To extract residual standard deviations, it would be useful to have a `sigma()` method (from the stats package).

**Comment by mattfidler** [comment]
I could see the use.  For me, I need to fix many other things first.  I will keep it on the list.

**Comment by billdenney** [comment]
Fair enough.  I may make a PR for it because I *think* it could be straightforward.

**Comment by mattfidler** [comment]
It should be;  In general the `fit$uif` has the parsed information and `fit$uif$ini` will have the bounds.  From there the `sd` is stored in `fit$theta`, so it should be extractable.

**Comment by mattfidler** [comment]
The broom methods work with nlmixr, so `tidy()` may fit your needs?  I can't recall if it separates these values or not.

**Comment by billdenney** [comment]
The rationale for a sigma method is for a simple test of the residual error
for categorization of results as having high, medium, or low variability.
Both `broom` and `fixef` can get me there, but sigma is a smidge simpler.
(Overall, this is very minor.)
